pkg/controllers/unifiedauth/unified_auth_controller.go

```diff
@@ -63,6 +63,11 @@ func (c *Controller) Reconcile(ctx context.Context, req controllerruntime.Reques
 		return controllerruntime.Result{}, nil
 	}
 
+	if cluster.Spec.ImpersonatorSecretRef == nil {
+		klog.Infof("cluster %s does not have an impersonator secret", cluster.Name)
```

Comment:
Much better now.
```suggestion
		klog.Infof("Aggregated API feature is disabled on cluster %s as it does not have an impersonator secret", cluster.Name)
```
We should give a more detail description of the effect.

Reply:
done